nasa-gibs/worldview
# WV-2970: Create E2E test for classification toggle feature

## Description
Some layers, like [MODIS Flood](https://worldview.earthdata.nasa.gov/?l=Coastlines_15m,MODIS_Combined_Flood_2-Day(disabled=3),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2023-12-07-T18%3A49%3A23Z), allow for classifications to be turned on/off by the user. Find the layer in the layer list, select View Options & Enable/Disable certain colors by clicking the toggles. The update should happen immediately.

Create an E2E test to capture this workflow.

Fixes #WV-2970

## How To Test
git checkout wv-2970
run `npx playwright test classification-test.spec.js` & confirm the classification tests pass
run `npx playwright test ` & confirm all of the tests pass

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
